authorization_feature.go

```diff
@@ -36,7 +36,31 @@ func (f *AuthorizationFeature) iAmIdentifiedAs(username string) error {
 	return nil
 }
 
+func (f *AuthorizationFeature) zebedeeRecognisesTheServiceAuthTokenAsInvalid() error {
```

Comment:
Im being uber nit picky now but something like `zebedeeDoesNotRecognisesTheServiceAuthToken()` feels nicer?

Feel free to ignore this comment 😅

Reply:
Yes I agree - changed